extension/src/tree/index.ts

```diff
@@ -61,6 +62,11 @@ export const getCliErrorTreeItem = (
     command: RegisteredCommands.EXTENSION_SHOW_OUTPUT,
     title: 'Show DVC Output'
   }
+
+  if (!isLabelPath) {
+    treeItem.label = label
```

Comment:
[C] If this is the only difference for the boolean flag why not create a wrapper function that calls `getCliErrorTreeItem` and updates the label of the resulting `treeItem`? The wrapper can get a descriptive name.